backend/migrator/migration/prod/2.7/0000##migrate_sensitive_data_policy_to_proto_json.sql

```diff
@@ -0,0 +1,55 @@
+-- Strip the NULL values.
+UPDATE policy SET payload = jsonb_strip_nulls(payload) WHERE type = 'bb.policy.sensitive-data';
+
+-- Delete the maskType field for each elements in array.
+UPDATE 
+  policy 
+SET 
+  payload = jsonb_set(
+    payload, 
+    '{sensitiveDataList}', 
+    (
+      SELECT 
+        jsonb_agg(item - 'maskType') 
```

Comment:
What's the meaning of `item - 'maskType'`

Reply:
> Delete key/value pair or string element from left operand. Key/value pairs are matched based on their key value.
For example: 
```sql
SELECT '{"a": "b", "c": 1}'::jsonb - 'a';
+----------+
| ?column? |
|----------|
| {"c": 1} |
+----------+
```